includes/Abstracts/Abstract_Ability.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Abstract Ability base class.
+ *
+ * @package WordPress\AI\Abstracts
+ */
+
+namespace WordPress\AI\Abstracts;
+
+use WP_Ability;
+
+/**
+ * Base implementation for a WordPress Ability.
+ *
+ * @since 0.1.0
+ */
+abstract class Abstract_Ability extends WP_Ability {
+
+	/**
+	 * The Feature class that the ability belongs to.
+	 *
+	 * @since 0.1.0
+	 * @var \WordPress\AI\Abstracts\Abstract_Feature
+	 */
+	protected $feature;
```

Comment:
Related to https://github.com/WordPress/ai/pull/61/files#r2491071832, I think it's premature to enforce a one-to-one relationship between an Ability and the module(s) that may consume it.

Reply:
Agreed, let's not couple them.